pyembed/Cargo.toml

```diff
@@ -17,6 +17,7 @@ links = "pythonXY"
 # Update documentation in lib.rs when new dependencies are added.
```

Comment:
Maybe I missed it if you already did it, but according to this comment, you need to update docs in `lib.rs`

Reply:
That's a good point, I checked out the lib.rs but I wasn't exactly sure what needed to be changed but I'll keep looking over the individual files to try to see what is needed for documentation.

In terms of which one it is allocating too I am actually setting the rust global allocator to mimalloc so the rust allocator should be mimalloc. This was enabled in Rust 1.28 [https://doc.rust-lang.org/edition-guide/rust-2018/platform-and-target-support/global-allocators.html]. Without mimalloc feature enabled it uses the normal system allocation and mimalloc is not compiled. When it is enabled I see it being compiled and should be set as the rust allocator, which I have cloned the pathway to use the raw_allocator="rust" from this point onward, though it would be nice to add test to make sure its enabled properly. 

In conclusion, it is the rust raw allocator except with the rust global allocator being set to mimalloc prior. Perceptually I've noticed a significant boost in app startup speed.